### PR TITLE
[DROOLS-1353] define ExecutableRunner interface - fix for osgi

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

File: kie-api/src/main/java/org/kie/api/runtime/RequestContext.java [hunk 1]
```diff
@@ -36,4 +36,12 @@ static RequestContext create() {
...
         }
     }
+    
+    static RequestContext create(ClassLoader classLoader) {
+        try {
+            return (RequestContext) Class.forName( "org.drools.core.command.RequestContextImpl", true, classLoader ).newInstance();
+        } catch (Exception e) {
+            throw new RuntimeException("Unable to instance RequestContext", e);
+        }
+    }
```
Comment (psiroky) [inline kie-api/src/main/java/org/kie/api/runtime/RequestContext.java]: Could we remove the duplication and call the other method? E.g. `create()` would call `create(ClassLoader)` with the default classloader.

**Comment by mswiderski** [comment]
makes sense, will update it soon

**Comment by mswiderski** [comment]
updated

**Comment by mswiderski** [comment]
any other comments or ready to merge?

**Pull request merged**